ans-abstract/abstract.tex

```diff
@@ -130,12 +130,12 @@ \subsection{Iodine-Sulfur Thermochemical Cycle}
 \section{Methodology}
 \label{method}
 
-A gasoline gallon equivalent (GGE) is the amount of fuel that can generate equivalent energy to a gallon of gasoline. One kilogram of hydrogen is equivalent to one gallon of gasoline \cite{noauthor_hydrogen_nodate}. Burning a gallon of gasoline produces 19.64 lbs of CO$_2$ \cite{noauthor_how_2014}. 
-Similarly, a diesel gallon equivalent (DGE) has the same amount of energy as a gallon of diesel. Approximately, a DGE is 113\% of a GGE \cite{noauthor_fuel_2014}, then 1.13 kg of hydrogen is equivalent to one gallon of diesel.
-A gallon of diesel produces 22.38 lbs of CO$_2$ \cite{noauthor_how_2014}. 
+A gasoline gallon equivalent (GGE) is the amount of fuel that can generate equivalent energy to a gallon of gasoline. One kilogram of hydrogen is equivalent to one gallon of gasoline \cite{doe_office_of_energy_efficiency_and_renewable_energy_hydrogen_2020}. Burning a gallon of gasoline produces 19.64 lbs of CO$_2$ \cite{us_energy_information_administration_how_2014}. 
```

Comment:
No change needed here, but I recommend the use of the glossaries package and an acros file to ensure that you always expand an acronym on the first use, and just the first use. Since you're already using those capabilities elsewhere in the file.... why not here? Inconsistency might get you in a bit of trouble some time.

Reply:
You are totally right. I tried to address that concern in the commit 7860b2e.